doc/source/news.rst

```diff
@@ -7,6 +7,238 @@
+
+  * We can load data like ``"key" : "[ { "key" : "value", ..., "key" : "value" } ]"`` as below.
+
+    .. code-block::
+
+      table_create Purchases TABLE_NO_KEY
+      column_create Purchases item COLUMN_SCALAR ShortText
+      column_create Purchases price COLUMN_SCALAR UInt32
+
+      table_create Settlements TABLE_HASH_KEY ShortText
+      column_create Settlements purchases COLUMN_VECTOR Purchases
+      column_create Purchases settlements_purchases COLUMN_INDEX Settlements purchases
+
+      load --table Settlements
+      [
+      {
+        "_key": "super market",
+        "purchases": [
+           {"item": "apple", "price": 100},
+           {"item": "milk",  "price": 200}
+        ]
+      },
+      {
+        "_key": "shoes shop",
+        "purchases": [
+           {"item": "sneakers", "price": 3000}
+        ]
+      }
+      ]
+
+  * By this feature, we can load data of JSON type almost as it is.
+  * Currently, this feature only support with JSON input.
+
+* Added support for UNIX epoch time for ``time_classify_*`` functions.
+
+  * We can use UNIX epoch time in ``time_classify_*`` functions as below.
+
+    .. code-block::
+
+       plugin_register functions/time
+
+       table_create Timestamps TABLE_PAT_KEY Time
+       load --table Timestamps
+       [
+       {"_key": 0},
+       {"_key": "2016-05-06 00:00:00.000001"},
+       {"_key": "2016-05-06 23:59:59.999999"},
+       {"_key": "2016-05-07 00:00:00.000000"},
+       {"_key": "2016-05-07 00:00:00.000001"},
+       {"_key": "2016-05-08 23:59:59.999999"},
+       {"_key": "2016-05-08 00:00:00.000000"}
+       ]
+
+       select Timestamps \
+         --sortby _id \
+         --limit -1 \
+         --output_columns '_key, time_classify_day_of_week(_key)'
+       [
+         [
+           0,
+           0.0,
+           0.0
+         ],
+         [
+           [
+             [
+               7
+             ],
+             [
+               [
+                 "_key",
+                 "Time"
+               ],
+               [
+                 "time_classify_day_of_week",
+                 null
+               ]
+             ],
+             [
+               0.0,
+               4
+             ],
+             [
+               1462460400.000001,
+               5
+             ],
+             [
+               1462546799.999999,
+               5
+             ],
+             [
+               1462546800.0,
+               6
+             ],
+             [
+               1462546800.000001,
+               6
+             ],
+             [
+               1462719599.999999,
+               0
+             ],
+             [
+               1462633200.0,
+               0
+             ]
+           ]
+         ]
+       ]
+
+* Improve performance for static index construction.
+
+  * Static index construction creates temporary tables internally.
+    Groonga truncate them after index construction and reuse.
```

Comment:
We should use third person singular form:

Groonga truncate**s** them after index construction and reuse**s**.